code/modules/reagents/chemistry/machinery/chem_recipe_debug.dm

```diff
@@ -24,41 +24,69 @@
 	var/minorImpurity
 	///The count of reactions that resolve below 0.9 purity
 	var/majorImpurity
-	///If we failed to react this current chem so use a lower temp
+	///If we failed to react this current chem so use a lower temp - all reactions only
 	var/failed = 0
+	///If we're forcing optimal conditions
+	var/force_temp = FALSE
+	var/force_ph = FALSE
+	///Forced values
+	var/f_temp = 300
+	var/f_ph = 7
+	///Multiplier of product
+	var/vol_multi = 20
```

Comment:
Don't use abbreviated variable names, extend these.

Reply:
Your best bet is to make those `force_temperature` and `force_ph`, and rename the existing ones `should_force_temperature` and `should_force_ph` (to make it obvious they are booleans).

Reply:
Okay, I think I changed them all successfully.